conda/recipes/rapids_core_dependencies/meta.yaml

```
@@ -0,0 +1,41 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+
+{% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev').lstrip('v') + environ.get('VERSION_SUFFIX', '') %}
+{% set py_version=environ.get('CONDA_PY', 39) %}
+{% set cuda_version = '.'.join(environ.get('CUDA', '11.5').split('.')[:2]) %}
+{% set cuda_major=cuda_version.split('.')[0] %}
+
+package:
+  name: rapids_core_dependencies
+  version: {{ version }}
+
+source:
+  git_url: ../../..
+
+build:
+  number: {{ GIT_DESCRIBE_NUMBER }}
+  string: cuda{{ cuda_major }}_py{{ py_version }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
```

Actually I'm not sure if this makes sense to include the Python version in the build string. This package doesn't rely on Python at all.
```suggestion
  string: cuda{{ cuda_major }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
```

---

These example packages don't rely on Python, and thus don't include Python versions in their build strings.
- https://anaconda.org/conda-forge/libgcc-ng/files
- https://anaconda.org/rapidsai-nightly/librmm/files